### ISSUE TITLE
SHA256 Publication

### ISSUE THREAD

**Issue by MSFT-Gareth** [PSBicep/PSBicep#246]
Publication of the SHA256 hash for the Bicep executable releases on the GitHub site.

This request is to allow for admins/users to verify when they download the installer executable from GitHub that it's local hash matches the one published.

**Comment by bjompen** [comment]
Good point! We should 100% at least publish the sha here as well. 
Might also be possible to add a -Verify (or better yet, -DontVerify and default to always verify sha when installing.)
Have to look in to if it's possible to get it using git API.

**Comment by MSFT-Gareth** [comment]
Yes its not publicly published anywhere so I have no way to verify once I download the installer that its the legit copy.
Something similar to what the Git team do would be the request here: https://github.com/git-for-windows/git/releases/
